firmware/config/engines/benelli.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+ * @file benelli.cpp
+ *
+ * set engine_type 33
+ *
+ * @date Feb 13, 2024
+ * @author Michael Holzer, (c) 2024
+*/
+
+#include "pch.h"
```

Comment:
I would like to avoid pre-cooked presets for rare/unusual engines, as they bloat the firmware image size and have potential for tech debt headaches later. Is it possible to base your test off of a pre-existing configuration?

Reply:
I fully understand the concern but there seems no pre-existing engine for 3-cylinder high rev motor cycles.
Yamaha, Triumph, MV Agusta and Benelli share technical details in  their motor cycle engine designs therefore this could be the foundation for a new class of engines.
I thought only this approach would assure proper testing for unusual engines early in the dev cycle.

Happy to use a different approach but would need hints/pointers

Reply:
As far as the ECU is concerned, there isn't anything special about these engines, or even odd-cylinder-count engines in general. It mostly treats any engine as N 1-cylinder engines, instead of 1 N-cylinder engine.

Reply:
I hear what you say, just wanted to make sure a high-rev engine gets tested with a unit test.
I didn't find an existing unit test going up to 12k RPM, therefore I tried to close this gap using a new engine type. I'm more than happy to implement the requirement using a different approach.

Please advise what approach would be a better solution from your point of view?

Reply:
https://github.com/rusefi/rusefi/commit/d3461e39051b070adeeb451d64a036a062cbc01f sounds like it could be related to my requirement but I lack the technical understanding to be sure

Reply:
That was fixed long ago and hasn't been a problem in FOME for a long time.